docs/current/sailbot_workspace/launch_files.md

```diff
@@ -0,0 +1,175 @@
+# ROS Launch Files in Sailbot Workspace
+
+ROS 2 Launch files allow us to programatically start up and configure multiple ROS nodes.[^1]
+Within Sailbot Workspace, ROS launch files are used to start up our ROS packages with ease.
+Additionally, we take advantage of the hierarchical properties of launch files by defining a global
+entry point that invokes the launch files of all ROS packages in the system.
+
+## Launch File Architecture
+
+There are two launch processes that we utilize: namely the Global Launch Process and the Package Launch Process.
+
+!!! tip "Interacting with the diagram"
+
+    - To switch between the global process and package launch process diagrams, hover over the image to make
+      the toolbar visible and navigate with the arrows on the left side
+
+<!-- This was generated by going to the source, selecting File > Embed > iFrame, and deselecting Layers, Tags, and Edit -->
+<!-- Source: https://drive.google.com/file/d/1B5ayOHnFcVvFmLeAGr6nOryFvVqk5abI/view?usp=sharing -->
+<iframe frameborder="0" style="width:100%;height:600px;" src="https://viewer.diagrams.net/?highlight=0000ff&nav=1&title=Launch_File_Architecture.drawio#Uhttps%3A%2F%2Fdrive.google.com%2Fuc%3Fid%3D1B5ayOHnFcVvFmLeAGr6nOryFvVqk5abI%26export%3Ddownload"></iframe>
```

Comment:
Some things that I think would make this easier to read
- Put the corresponding launch process in each section
- Label the diagram components with the numbers used in the description
- Go over package first, then global as a diff/extension to package